sdk/sdk-core/aar/src/main/java/mil/emp3/core/storage/StorageManager.java

```diff
@@ -917,6 +917,26 @@ private void processBulkFeatureApplyList(Object userContext) {
         //Log.d(TAG, "Processed List.");
     }
 
+    /**
+     * This apply method is called by all the feature editors.  The editors already
+     * have a map instance and, because they are drawing or editing, the feature
+     * is visible. It saves processing of going through the steps to find the values
+     * of mapInstance and visible, as in the second apply method below.  Also the
+     * editor changes apply immediately and are never processed in a batch.
+     *
+     * @param mapInstance current instance where editor is drawing
+     * @param feature     feature to apply
+     * @param userContext any object from user
+     */
+
```

Comment:
remove extra space

Reply:
done

Reply:
The space is still there, unless I am missing something...